src/common/models.py

```diff
@@ -3,11 +3,30 @@
 from account.models import Profile
 
 
+class Votes(models.Model):
+        upvoters = models.ManyToManyField(
+            Profile,
+            related_name="upvotes",
+        )
+        downvoters = models.ManyToManyField(
+            Profile,
+            related_name="downvotes",
+        )
+
+        def __str__(self):
+            return ''.join([str(self.comment_item), '\'s votes'])
```

Comment:
This won't work. You need to create a function that returns question_comment_item, answer_comment_item, or comment_comment_item, whichever one is not None, then call that here.

Reply:
Probably fixed in https://github.com/DevTeamSCH/vikoverflow-backend/pull/11/commits/c70fc16f0b3b8ae26850087c89ea6c1c74c87454, not tested yet.